### ISSUE TITLE
Unit Test Failed all

### ISSUE THREAD

**Issue by alariver** [preset-io/elasticsearch-dbapi#21]
## Using docker-composer.yml in repository( using docker run Elasticsearch 7.3.2), Unit test failed...
```
test_data_data1 (es.tests.test_0_fixtures.TestData) ... ok
test_data_flights (es.tests.test_0_fixtures.TestData) ... ok
DBAPI: test Elasticsearch is called with user password ... ok
DBAPI: Test boolean description ... ok
DBAPI: Test connection failed ... ok
DBAPI: Test commit method exists ... ok
DBAPI: Test connection failed ... ok
DBAPI: Test datetime description ... ok
DBAPI: Test execute query with no results ... ok
DBAPI: Test execute and fetchall ... ok
DBAPI: Test execute and fectchmany ... ok
DBAPI: Test execute and fectchone ... ok
DBAPI: Test execute, fetchall on connect ... ok
DBAPI: Test execute and rowcount ... ok
DBAPI: Test execute select all (*) ... ok
DBAPI: Test execute select with wrong table ... ok
DBAPI: Test executemany not supported ... ok
DBAPI: test Elasticsearch is called with https ... ok
DBAPI: Test number description ... ok
DBAPI: Test simple group by ... ok
DBAPI: Test string description ... ok
SQLAlchemy: test Elasticsearch is called with user password ... ERROR
SQLAlchemy: test Elasticsearch is called with https and param ... ERROR
SQLAlchemy: test Elasticsearch is called with https and param ... ERROR
SQLAlchemy: test Elasticsearch is called with advanced config params ... ERROR
SQLAlchemy: test Elasticsearch with param value error ... ERROR
SQLAlchemy: test Elasticsearch is called for multiple hosts ... ERROR
SQLAlchemy: Test execute select with wrong table ... ERROR
SQLAlchemy: Test get_columns exclude arrays ... ERROR
SQLAlchemy: Test get schema names ... ERROR
SQLAlchemy: Test has_table ... ERROR
SQLAlchemy: Test get_columns ... ERROR
SQLAlchemy: Test reflection get_tables ... ERROR
SQLAlchemy: Test select all ... ERROR
SQLAlchemy: Test simple query ... ERROR
```
## pip freeze
```
% pip freeze | grep -i sql
Flask-SQLAlchemy==2.4.1
marshmallow-sqlalchemy==0.22.2
SQLAlchemy==1.3.13
SQLAlchemy-Utils==0.36.1
sqlparse==0.3.0
% pip freeze|grep -i elasticsearch
elasticsearch==7.5.1
elasticsearch-dbapi==0.1.1

```

**Comment by dpgaspar** [comment]
Use the same `elasticsearch` from this repo requirements.txt or version 7.1.0

**Comment by alariver** [comment]
Got it!
`pip install elasticsearch-dbapi` cause the elasticsearch 7.6.0 was installed.

**Comment by dpgaspar** [comment]
Released elasticsearch-dbapi version 0.1.2, now you can use a newer elasticsearch version without issues
